apps/ima/translations/src/en/validation.json

```diff
@@ -75,6 +75,25 @@
   "has-address": {
     "required": "You must select an option"
   },
+  "house-number": {
+    "required": "Enter your house number/name, and street name",
+    "notUrl": "Please do not enter a link/url into your answers"
+  },
+  "street": {
+    "notUrl": "Enter a valid street name"
```

Comment:
This should be "Please do not enter a link/url into your answers" for `"notUrl"`